src/ContextMenuTrigger.js

```diff
@@ -51,11 +51,10 @@ export default class ContextMenuTrigger extends Component {
         if (this.props.holdToDisplay >= 0) {
             event.persist();
 
-            this.touchHandled = true;
-
             this.touchstartTimeoutId = setTimeout(
                 () => this.handleContextClick(event),
-                this.props.holdToDisplay
+                this.props.holdToDisplay,
+                this.touchHandled = true
```

Comment:
Woops, yeah, this is you should add curly brackets and semicolons as soon as you start doing more than one thing.